src/Components/ImportModal/ImportModal.module.css

```diff
@@ -0,0 +1,124 @@
+.renderMedia {
+    width: 100%;
+    height: auto; 
+    object-fit: contain;
+  }
+  .canvaHomePage {
+    background-color: rgb(250, 250, 250);
+    min-height: 90vh;
+    width: 100vw;
+    margin: 0 auto;
+    background-size: cover;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+  }
+  
+  .dropModal {
+    background-color: rgb(0, 196, 204, 50%);
```

Comment:
Would be good to set up variables for these colours

Reply:
yepp will do, only used the canva brand colors. I will make this change at a later PR